autobahn/websocket/protocol.py

```diff
@@ -3394,7 +3394,7 @@ def processProxyConnect(self):
             # HTTP version
             #
             http_version = sl[0].strip()
-            if http_version != "HTTP/1.1":
+            if http_version not in ("HTTP/1.1", "HTTP/1.0"):
                 return self.failProxyConnect("Unsupported HTTP version ('%s')" % http_version)
```

Comment:
If we're going to support proxies, we'll need to support HTTP/1.0. A whole bunch of proxies (Squid, as mentioned, for example) only support HTTP/1.0, until very recently.